src/main/java/teammates/common/datatransfer/FeedbackResponseCommentAttributes.java

```diff
@@ -191,9 +191,7 @@ public String getJsonString() {
     @Override
     public void sanitizeForSaving() {
         this.commentText = Sanitizer.sanitizeTextField(this.commentText);
```

Comment:
It seems like we already have the function for this purpose, after all. See if this function differs from the newly defined `sanitizeForRichText`, and if we can decommission one of them (i.e just use one function for all purposes).